helm/reana/templates/cronjobs.yaml

```diff
@@ -4,6 +4,7 @@ apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: {{ include "reana.prefix" . }}-system-status
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think we also need `REANA_INFRASTRUCTURE_KUBERNETES_NAMESPACE` `REANA_RUNTIME_KUBERNETES_NAMESPACE` here, otherwise it can't reach the DB properly.

Reply:
Absolutely, thanks for the heads up (we need https://github.com/reanahub/reana/issues/277 to make this easier).